Sources/Core/Filters/Root/RootFilterViewController.swift

```diff
@@ -98,9 +88,9 @@ extension RootFilterViewController: UITableViewDataSource {
 
         switch currentFilter.kind {
         case .search:
-            let cell = tableView.dequeue(SearchQueryCell.self, for: indexPath)
-            cell.searchBar = searchQueryViewController.searchBar
-            cell.searchBar?.placeholder = currentFilter.title
+            let cell = tableView.dequeue(FreeTextFilterCell.self, for: indexPath)
+            cell.configure(with: freeTextFilterViewController.searchBar)
+            freeTextFilterViewController.setup(with: currentFilter)
```

Comment:
It's nice we got rid of `searchFilter` property, but maybe we can create `FreeTextFilterViewController` with filter here? Then we don't have to unwrap optional `filter` in `FreeTextFilterViewController`.

Reply:
I can do that :)